### PR TITLE
CBBD-124 parse and transform Hospice claims

### PR DESCRIPTION
**Pull request by dho937**

### CODE REVIEW

**Comment by dho937** [comment]
@karlmdavis @sbrockhoff  Please review when you get a minute.  Thanks.

File: bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java [hunk 1]
```diff
@@ -1501,6 +1503,129 @@ public void transformInsertSNFClaimEvent() {
...
+	 */
+	@SuppressWarnings({ "rawtypes", "unchecked" })
+	@Test
+	public void transformInsertHospiceClaimEvent() {
+		// Create the mock bene to test against.
+		HospiceClaimGroup record = new HospiceClaimGroup();
+		record.version = RifFilesProcessor.RECORD_FORMAT_VERSION;
+		record.recordAction = RecordAction.INSERT;
+		record.beneficiaryId = "42";
+		record.claimId = "SuttersMill";
```
Comment (sbrockhoff) [inline bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java]: Any idea why we are using "SuttersMill" for the claim id for several of the claim tests? Per the data dictionary this data will be numeric (even if the data type isn't technically a number)

Comment (dho937) [inline bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java]: Just was in there from prior code.  I can change to be a valid claim number.

**Comment by sbrockhoff** [comment]
:+1:

File: bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java (hunk 1 above)
Comment (karlmdavis) [inline bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java]: Oh... there goes my (not very funny) joke. https://en.wikipedia.org/wiki/Sutter%27s_Mill, a goldrush "claim".

Bah humbug, I say!

**Comment by karlmdavis** [comment]
Belatedly: :+1:

**Pull request merged**